twine/commands/check.py

```diff
@@ -68,47 +68,81 @@ def __str__(self):
         return self.output.getvalue()
 
 
+def _check_file(filename, render_warning_stream):
+    """Check given distribution.
```

Comment:
I thought we had flake8-docstrings on this repo, but it would appear we don't. Please make the docstring a single line. I'll work on adding that to our linters.

Reply:
I don't like that style but I'm happy to make this change. :)